src/tests/Interop/MarshalAPI/FunctionPointer/GenericFunctionPointer.cs

```diff
@@ -40,6 +46,11 @@ internal static unsafe BlittableGeneric<T> WrappedGenericCalli<U>(void* fnptr, U
         {
             return ((delegate* unmanaged<U, BlittableGeneric<T>>)fnptr)(arg);
         }
+
+        internal static unsafe void NonGenericCalli<U>(void* fnptr, ref int val, float arg)
+        {
+            ((delegate* unmanaged<float, ref int, void>)fnptr)(ref val, arg);
```

Comment:
```suggestion
            ((delegate* unmanaged<ref int, float, void>)fnptr)(ref val, arg);
```

If the arguments should be the other way, we might want a comment.